package.json

```diff
@@ -15,23 +16,33 @@
   "author": "Brian Leathem",
   "license": "MIT",
   "dependencies": {
+    "connect-mongo": "^1.3.2",
+    "connect-redis": "^3.1.0",
     "express": "4.14.0",
+    "express-session": "^1.14.2",
     "fh-mbaas-api": "5.14.2",
+    "fh-wfm-mediator": "0.1.0-rc1",
+    "fh-wfm-simple-store": "0.0.4",
     "lodash": "4.7.0",
+    "mongodb": "^2.2.16",
     "q": "1.4.1",
+    "redis": "^2.6.3",
     "semver": "^5.3.0",
-    "shortid": "^2.2.6",
-    "fh-wfm-mediator": "0.0.15"
+    "shortid": "^2.2.6"
   },
   "devDependencies": {
     "body-parser": "^1.15.2",
+    "cookie-parser": "^1.4.3",
```

Comment:
These were here for some old unit(/integration?) tests with `supertest` that were broken and thus removed for now, I'll go ahead and clean up these dependencies too.